LPD-58579

davisaints opened the pull request:
**Related ticket: [LPD-58579](https://liferay.atlassian.net/browse/LPD-58579)**

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-source-format/8357/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/8357/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-source-format/8357">test-portal-source-format#8357</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/8357/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/natocesarrego/liferay-portal/pull/1447">natocesarrego#1447</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/253986123">[master] ci:test:sf - davisaints &gt; natocesarrego - PR#1447 - 2025-07-02[03:45:22]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>253986123</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-source-format/8357/">test-portal-source-format#8357</a><br/></div></div>

davisaints commented:
Changed

```
test(?:\\.skip)?\\(\\s*(['\"`])(.*?)\\1
```

to 

```
`test(\\.skip)?\\(\\s*(['\"`])(.*?)\\2`
```

The original regex used a non-capturing group for `.skip`, so it didn’t tell us if the test was skipped. I changed it to a capturing group so we can detect and capture `.skip` when it's present.

natocesarrego commented:
Manually forwarded to Brian: https://github.com/brianchandotcom/liferay-portal/pull/163334
